roles/static_inventory/tasks/checkpoint.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: create static inventory dir
+  file:
+    path: "{{ inventory_path }}"
```

Comment:
Yeah. Plus replacing inventory_path with inventory_dir because that's a special variable that points to the current inventory.